body.tex

```diff
@@ -180,29 +181,28 @@ \section{Long Term Planning}
     \item
         A set of \emph{Milestones}, each of which represents the delivery or availability of specific functionality. Each planning package culminates in a milestone, and may contain other milestones describing intermediate results.
     \item
-        A series of \emph{epics}, which describe major pieces of technical work. Epics are associated with concrete, albeit high-level, deliverables in the \gls{shape} of the above milestones, and have specific resource loads (staff assignments), start dates, and durations.
+        A series of \emph{epics} describe major pieces of work. Epics are associated with concrete, albeit high-level, deliverables in the \gls{shape} of the above milestones, and have specific resource loads (staff assignments), start dates, and durations.
 \end{enumerate}
 
-Milestones are allocated to one of four levels, defined as follows:% PJM: Can we not design a scheme where we use the term "level" to label both the WBS activities AND the milestones? We seem so close! As it is, though, I think we have to write the following text:
-\footnote{Note that the level of the milestone is logically distinct from the level of the WBS and OBS, even though the same term (``level'') is used to label both. In the OBS, level 0 is the Observatory, level 1 is the departments, level 2 is the teams, and level 3 is the groups within the teams. We take care to always disambiguate ``WBS level'' from ``milestone level'' and never use the term ``level'' without these qualifiers preceding it.}
+Milestones are allocated to one of four levels, defined as follows:
 
 \begin{description}
-%this is not consistent with NOIRLab or our current FY21 plan. We need to discuss an decide how to capture these. Could be one or two RDO level MS and several L2 MS for each department both of which show up in the POP. Or we keep the Major MS classification and have several of those in each Dep. Suppose we had one or two L1 milestones like Annual DR. Then all the current Major MS in our sheet could be L2s that point at that.
 \item[Level 1]
-These are at the full observatory level owned by the directors office
+These are at the full observatory level and owned by the Directors Office. 
+An example is the completion of an annual Data Release.
 \item[Level 2]
-These reflect cross department commitments. As such, they must be defined
-in consultation with the Director's Office
+These reflect effort within a Department and are owned by an Associate Director, or are cross-Department commitments. 
```

Comment:
I think cross-department commitments ALSO need to be owned by a single AD (one neck).